app/dao/notifications_dao.py

```diff
@@ -588,13 +584,17 @@ def dao_get_notifications_by_to_field(service_id, search_term, notification_type
             normalised = search_term.lower()
 
     else:
-        raise InvalidRequest("Only email and SMS can use search by recipient", 400)
+        raise InvalidRequest("Must specify email or sms to search by recipient", 400)
```

Comment:
The error message sounds vague. I think:
"Searching by recipient is only available for sms and email messages"

Would work better.

Also earlier notification_type defaulted to None, now it does not have a default value. Is it possible this change of behaviour would break something?

Reply:
Changed the behaviour to accept `None` for notification type because users without the _See dashboard_ permission can still search across emails and text messages (they see both on one screen).